types/setting.go

```diff
@@ -195,6 +199,28 @@ var (
 		SettingNameBackingImageCleanupWaitInterval:              SettingDefinitionBackingImageCleanupWaitInterval,
 		SettingNameGuaranteedEngineManagerCPU:                   SettingDefinitionGuaranteedEngineManagerCPU,
 		SettingNameGuaranteedReplicaManagerCPU:                  SettingDefinitionGuaranteedReplicaManagerCPU,
+		SettingNameSupportBundleImage:                           SettingDefinitionBundleImage,
```

Comment:
Wrong naming pattern. Should be `SettingDefinitionSupportBundleImage`

Reply:
Updated